tur/yc/build.sh

```diff
@@ -0,0 +1,59 @@
+TERMUX_PKG_HOMEPAGE=https://git.yzena.com/Yzena/Yc
+TERMUX_PKG_DESCRIPTION="Monopackage with Rig, Yao, and Yvm"
```

Comment:
It might be worth splitting out the individual programs of the `yc` multi-call binary,
into [subpackages](https://github.com/termux/termux-packages/wiki/Creating-new-package#writing-a-subpackage-script) which then create their respective links in `$PREFIX/bin`, or create a simple wrapper shell script to call the main `yc` binary in the correct manner.

This may be more flexible than a simple `ln -sf` in `termux_make_install()`.
For a *similar-ish* example see: https://github.com/termux/termux-packages/blob/37e77ab30f722ade25b9a24c8ffed077b083c356/packages/libllvm/libcompiler-rt.subpackage.sh#L18-L40

Reply:
Making a subpackage was confusing, and I don't know if I did it correctly, but it doesn't give any warnings.